policies/recipes/steps.go

```diff
@@ -370,51 +372,60 @@ func extractTar(tarName string, dst string, archiveType osconfigpb.SoftwareRecip
 }
 
 // StepMsiInstallation builds the command for a MsiInstallation step
-func StepMsiInstallation(step *osconfigpb.SoftwareRecipe_Step_MsiInstallation, artifacts map[string]string) error {
-	fmt.Println("StepMsiInstallation")
-	return nil
+func StepMsiInstallation(step *osconfigpb.SoftwareRecipe_Step_InstallMsi, artifacts map[string]string, runEnvs []string, stepDir string) error {
```

Comment:
Can we name the functions after teh steps, so StepInstallMSI, and StepInstalDPKG